File: bubble.js
function bubbleSort(a) {
    for (let i = 0; i < a.length - 1; i++) {
        for (let j = 0; j < a.length - i - 1; j++) {
            if (a[j] > a[j + 1]) {
                let temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }
}

function printArr(a) {
    console.log(a.join(" "));
}

let a = [70, 15, 2, 51, 60];
console.log("Antes de ordenar:");
printArr(a);
bubbleSort(a);
console.log("Después de ordenar:");
printArr(a);

File: bubble.cpp
#include <iostream>
using namespace std;

void bubbleSort(int a[], int n) {
    for (int i = 0; i < n - 1; i++) {
        for (int j = 0; j < n - i - 1; j++) {
            if (a[j] > a[j + 1]) {
                int temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }
}

void printArr(int a[], int n) {
    for (int i = 0; i < n; i++)
        cout << a[i] << " ";
    cout << endl;
}

int main() {
    int a[] = {70, 15, 2, 51, 60};
    int n = sizeof(a) / sizeof(a[0]);
    cout << "Antes de ordenar:" << endl;
    printArr(a, n);
    bubbleSort(a, n);
    cout << "Después de ordenar:" << endl;
    printArr(a, n);
    return 0;
}
